### PR TITLE
AZURE ACR - Fixing login not working with admin access

### PR DESCRIPTION
**Pull request by o-reo**

### CODE REVIEW

File: Babylon/groups/azure/groups/acr/registry_connect.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    response = subprocess.run(["az", "acr", "login", "--name", registry_part, "--expose"],
+                              shell=False,
+                              check=False,
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE)
+    decoded = json.loads(response.stdout)
+    # Login to docker with the returned access token
+    client = docker.from_env()
+    try:
+        client.login(username="00000000-0000-0000-0000-000000000000",
```
Comment (johnfolly) [inline Babylon/groups/azure/groups/acr/registry_connect.py:28]: username=resgistry also

**Review by johnfolly — APPROVED** [review]
Thank you @o-reo ,

It perfectly working now 👌

File: Babylon/groups/azure/groups/acr/registry_connect.py (hunk 1 above)
Comment (o-reo) [reply Babylon/groups/azure/groups/acr/registry_connect.py:28]: In fact username is literally 000, it does not work with username registry

Comment (johnfolly) [reply Babylon/groups/azure/groups/acr/registry_connect.py:28]: Ah Ok  😅 !

**Pull request merged**